engine/pom.xml

```diff
@@ -89,6 +89,12 @@
             <artifactId>paranamer</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>javax.money</groupId>
+            <artifactId>money-api</artifactId>
+            <version>1.0.1</version>
```

Comment:
Could you add properties for these versions in the parent POM and add them to the `dependencyManagement` block in there?